abc.templates/default-workflows/contents/guardian-plan.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: 'actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9' # ratchet:actions/checkout@v3
         with:
           fetch-depth: 0 # get everything so we can use git diff
-          ref: '${{ github.event.pull_request.head.ref }}'
+          ref: '${{ github.event.pull_request.head.sha }}'
```

Comment:
I think you should update the usages in the .github/workflow/ files too

Reply:
Done.